resources/public/api/conf/1.0/retrieveCodingOutUnderpayments.yaml

```diff
@@ -0,0 +1,102 @@
+
+description: |
+  This endpoint allows the developer to retrieve HMRC held coded out underpayments and coded out debt amounts or user submitted amended coded out underpayments and coded out debt amounts for a previous tax year and given NINO.
```

Comment:
I don't think this matches the RAML exactly but it also seems like a slightly clearer description. Where is it from?

Reply:
I got it from the [spec](https://confluence.tools.tax.service.gov.uk/pages/viewpage.action?pageId=256479026). Shall I change the RAML to match?

Reply:
I think it was agreed at one of the stand-ups that we would make the OAS to match the RAML. And then if we found any places where the RAML was not in-sync with the specs (such as now) we would make a ticket to revisit it and fix the raml + oas then. 

Please correct me if that's not right!

Reply:
Cool ill change the OAS to match the RAML now